src/Control/Effect/Lift.hs

```diff
@@ -30,4 +33,20 @@ import Control.Effect.Lift.Internal (Lift(..))
 --
 -- @since 1.0.0.0
 sendM :: (Has (Lift n) sig m, Functor n) => n a -> m a
-sendM = send . Lift . fmap pure
+sendM m = send (LiftWith (\ ctx _ -> (<$ ctx) <$> m) pure)
+
+
+-- | Run actions in an outer context.
+--
+-- This can be used to provide interoperation with @base@ functionality like 'Control.Exception.catch':
```

Comment:
Is it worth mentioning the dreaded dropped-writes problem? Probably not, but maybe?

Reply:
That’s no more of a problem with `catch` than with `catchError`, so I don’t think it’s worth calling out here specially.

Reply:
Ehhhh I’ve changed my mind.